native/cocos/platform/ios/modules/SystemWindow.mm

```diff
@@ -46,9 +47,10 @@ of this software and associated engine source code (the "Software"), a limited,
 }
 void SystemWindow::closeWindow() {
     // Force quit as there's no API to exit UIApplication
-    cc::events::Close::broadcast();
```

Comment:
Why delete `Close::broadcast();`?

Reply:
Before Close::broadcast() call I guess because the exit function is called directly to exit. 'Close::broadcast()' should be called in the release process